### ISSUE TITLE
Support a new testing realm/server

### ISSUE THREAD

**Issue by nchamo** [decentraland/explorer#1424]
As of now, working in preview/export mode has some limitations or differences with production:
* Multiplayer is not supported
* Asset bundles are not created or used
* Other scenes are not loaded

These differences can have a huge impact in the final user experience, so content creators are forced to push their changes to production and test them there. This is of course not ideal:
* Content creators lose the element of surprise
* Users may have a bad experience by interacting with a scene that is not really finished
* Content servers have to handle more load, since each deployment is propagated through all servers

A possible solution might be to have a separate Catalyst server to be used only for testing. We could:
* Deploy a new Catalyst server outside the DAO
* Configure it so that it uses a realm with the name `testing`
* Make sure it pushes to the Asset Bundle converter

If we go with this approach, then deployments made to this server would not be propagated to the other realms, but deployments made to other servers would also be published on this new testing server.

Then, on the explorer side, we would need to make sure that this realm is used only when the realm is specified in the url (ignore any caching). For this approach to work we would need to make some changes to the explorer, since only whitelisted & DAO-approved Catalysts are supported right now.

**Comment by eordano** [comment]
+1000

I'd parse `/?catalyst=example.com` to force this mode.

Another variations of this, that might be a different issue, is that I'd love to populate the default `/` HTTP route for Catalysts to be a Explorer version that only connects to that catalyst

**Comment by nchamo** [comment]
I would **love** to have it as simple as `?catalyst=example.com`!!!

And about the other feature you mention, I'll create an issue for it. Should be super simple and a really could thing to have

**Comment by nchamo** [comment]
I just did some research about this feature. Apparently, it is **almost** possible to connect to a custom catalyst. Assuming that your domain is this:
```domain = 'https://peer.decentraland.org'```

You would need to add the following query parameters to the explorer:
```
&FETCH_PROFILE_SERVICE=domain/lambdas/profile
&UPDATE_CONTENT_SERVICE=domain/content
&FETCH_CONTENT_SERVICE=domain/content
&FETCH_META_CONTENT_SERVICE=domain
&COMMS_SERVICE=domain/comms
&RESIZE_SERVICE=domain/lambdas/images
```

The only data that I found that can't be configured, it the realm information. Apparently, even if you connect to a custom catalyst, you still have the realm data that points to one of the whitelisted catalysts. 

So it looks like we would need to:
* Make it easier for users to use a custom catalyst by adding a `&catalyst=` parameter, instead of having to configure all services separately
* When a custom catalyst is set, don't check the cache or DAO to get the realm data, just use the given catalyst
